playbooks/openshift-glusterfs/private/uninstall.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Uninstall GlusterFS
+  hosts: oo_first_master
+  tasks:
+  - name: Run glusterfs uninstall role
+    include_role:
+      name: openshift_storage_glusterfs
+      tasks_from: uninstall.yml
```

Comment:
You've changed the file name here but not the actual file's name.

Reply:
Now, we have two files at roles/openshift_storage_glusterfs/tasks/  1. uninstall.yml  2. glusterfs_uninstall.yml

Reply:
uninstall.yml is the top level file there.